frontend/src/app/services/storage-handler.service.ts

```diff
@@ -0,0 +1,108 @@
+/**
```

Comment:
I am not sure if we should name the folder "services" or just "service". (Similar to "model" vs. "models") What do you think?

I just looked it up on the internet since I always used the singular form. Originally, I thought that naming the folders "model" and "service" would be part of the Angular Style Guide. But I actually didn't find it here. So I simply don't know where I got the ideas for naming the folders from. There are a few projects that do it in a relatively similar fashion (e.g. https://github.com/gothinkster/angular-realworld-example-app)

Reply:
I uses plural form if there gonna be more services under it. Currently it is not there, but in future we may have more services. I thought it is just similar to assets environments folder.